### DISCUSSION TITLE
Webpack v4 to Webpack v5 – loaders via CLI deprecation ?

### DISCUSSION THREAD

**Discussion by projektorius96** [webpack/webpack#14856] [Q&A] [answered]
**Webpack v4** says there are three ways to use loaders:
- webpack.config.js
- import
- _CLI_

[Docs for v5 ](https://webpack.js.org/concepts/loaders/)

**Webpack v5** says there are only two ways to use loaders:
- webpack.config.js
- import

[Docs for v4](https://v4.webpack.js.org/concepts/loaders/)

**My question**: Have you deprecated loaders via CLI in v5?

Thank you.

**Comment by alexander-akait** [comment, marked as answer]
Yes, loaders via CLI are deprecated

**Reply by projektorius96** [reply]
Thanks for answering . I assume this should be highlighted in docs , because it's a major transition between neighbour-like versions. ❗

**Reply by alexander-akait** [reply]
Feel free to open an issue in our docs :+1:

**Reply by projektorius96** [reply]
@alexander-akait [done](https://github.com/webpack/webpack.js.org/issues/5735)
